subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -3765,6 +3765,24 @@ static int lwm2m_engine_observation_handler(struct lwm2m_message *msg, int obser
 	return r;
 }
 
+static int lwm2m_engine_default_content_format(uint16_t *accept_format)
+{
+#if LWM2M_PROTOCOL_VERSION_MAJOR == 1 && LWM2M_PROTOCOL_VERSION_MINOR == 1
```

Comment:
I think it'd be cleaner to wait for #42169 and use `IS_ENABLED(CONFIG_LWM2M_VERSION_1_1)`

Reply:
OK I Update that Part. It not pass a compile process untill #42169 is merged.